LL/dal_corpri_arad.ttl

```diff
@@ -5321,9 +5321,14 @@
     rel:childOf <#NadBuidb-fb5b07ec>.
 
 <#NadBuidb-fb5b07ec>
-    owl:sameAs <#NadBuidb>;
+    a foaf:Person;
+    irishRel:nomName "Nad Buidb";
     rel:childOf <#Illaind>.
 
+# This Nad Buidb was marked as owl:sameAs <#NadBuidb> but I don't see
+# any reason for thinking so. The text presents them in close proximity
+# with different fathers. EPT
+
```

Comment:
I think that we should go with your interpretation.  We should probably put a comment in explaining the situation so that if someone wants to change it based on their own understanding of the text, they have that chance.